data/ui/Window.ui

```diff
@@ -8,6 +8,20 @@
       <object class="GtkOverlay">
         <property name="visible">True</property>
         <property name="can_focus">False</property>
+        <child>
+          <object class="GtkStack" id="stack">
+            <property name="visible">True</property>
+            <property name="can_focus">False</property>
+            <property name="transition_duration">400</property>
+            <property name="transition_type">slide-left-right</property>
+            <child>
+              <placeholder/>
+            </child>
+          </object>
+          <packing>
+            <property name="index">-1</property>
+          </packing>
+        </child>
```

Comment:
This is Glade moving things around upon save; if I revert it now it will only do it again next time I edit this file. I think we should let it through tbh.